io.github.hamza_algohary.Coulomb.yaml

```
@@ -0,0 +1,55 @@
+id: io.github.hamza_algohary.Coulomb
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: coulomb
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.openjdk
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: 
+      - /usr/lib/sdk/openjdk/install.sh
+  - name: jar
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/hamza-algohary/Coulomb/releases/download/v0.6/Coulomb.jar 
+        sha256: fcb29ae220f40952923b78dbf390e1798dabb2825b1dd114afd803124f09fe78
+    build-commands:
+      - mkdir -p /app/bin
+      - install -Dm 755 Coulomb.jar /app/bin        
+  - name: data
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/hamza-Algohary/Coulomb/raw/main/app/src/main/resources/icons/vector/dark/coulomb.svg
+        sha256: 4ef6efc7f09dbf0c37a0fb75c287df0a9fdde57f6df2f74eb6b3e987ec1b6615
+        dest-filename: io.github.hamza_algohary.Coulomb.svg
+      - type: file
+        url: https://github.com/hamza-algohary/Coulomb/raw/refs/heads/main/app/src/main/resources/io.github.hamza_algohary.Coulomb.desktop
+        sha256: f02424f897a640a5a78cfd9507032aee81b2d37e0d129dfb34f3f0727e182dc6
+      - type: file
+        url: https://github.com/hamza-algohary/Coulomb/raw/refs/heads/main/app/src/main/resources/io.github.hamza_algohary.Coulomb.metainfo.xml
+        sha256: d39c9f32efe6d921024e28fcd2d722ddeec31c8ada46f3745806dd6fa6a3c6ec
+      - type: script
+        dest-filename: coulomb
+        commands: 
+          - "#!/bin/bash"
+          - "java -jar /app/bin/Coulomb.jar"
+    build-commands:  
+      - install -Dm 644 io.github.hamza_algohary.Coulomb.svg -t /app/share/icons/hicolor/scalable/apps
+      - install -Dm 644 io.github.hamza_algohary.Coulomb.metainfo.xml -t /app/share/metainfo
+      - install -Dm 644 io.github.hamza_algohary.Coulomb.desktop -t /app/share/applications
+      - install -Dm 755 coulomb /app/bin
+
+
+finish-args: 
+  - --env=PATH=$PATH:/app/jre/bin:/app/bin:/usr/bin
```

```suggestion
  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
```

There is no `$PATH` in that context.